### PR TITLE
Eth upd tests  3rdPartyRewards

### PR DESCRIPTION
**Pull request by AlexChetverov**

### CODE REVIEW

File: test/parallel/proffOfStake.3rdPartyRewards.activation.test.ts [hunk 1]
```diff
@@ -136,8 +137,18 @@ describe("Proof of stake tests", () => {
         testUser1,
         await ProofOfStake.rewardPool(
           MGA_ASSET_ID,
-          newToken,
           newToken3,
+          newToken3,
+          Assets.DEFAULT_AMOUNT.muln(10e6),
```
Comment (goncer) [inline test/parallel/proffOfStake.3rdPartyRewards.activation.test.ts:142]: why this change? 
Before it was rewarding the pool of ( MGX , token ) with a token3, and now, it seems that it will be rewarding a pool of ( MGX , token3 ) with token3. that would mean that we are not strictly testing the same.  One is rewarding a pool with a token that does not belong to the pool, and the other is rewarding a pool with a token that belongs to it. 
where there any issue running the previous setup?

File: test/parallel/proffOfStake.3rdPartyRewards.activation.test.ts [hunk 2]
```diff
@@ -97,9 +98,9 @@ describe("Proof of stake tests", () => {
       Sudo.sudoAs(
         testUser3,
         Xyk.createPool(
-          newToken2,
+          MGA_ASSET_ID,
           Assets.DEFAULT_AMOUNT.muln(20e6),
-          newToken3,
+          newToken4,
           Assets.DEFAULT_AMOUNT.muln(20e6),
         ),
```
Comment (goncer) [inline test/parallel/proffOfStake.3rdPartyRewards.activation.test.ts:105]: why this change?

File: test/parallel/proffOfStake.3rdPartyRewards.activation.test.ts [hunk 3]
```diff
@@ -38,32 +34,37 @@ describe("Proof of stake tests", () => {
...
-    keyring = new Keyring({ type: "sr25519" });
-    sudo = new User(keyring, getEnvironmentRequiredVars().sudo);
+    sudo = getSudoUser();
     [testUser1, testUser2, testUser3] = setupUsers();
-    [newToken, newToken2, newToken3] = await Assets.setupUserWithCurrencies(
-      sudo,
-      [Assets.DEFAULT_AMOUNT, Assets.DEFAULT_AMOUNT, Assets.DEFAULT_AMOUNT],
-      sudo,
-      true,
-    );
```
Comment (goncer) [inline test/parallel/proffOfStake.3rdPartyRewards.activation.test.ts:50]: I think that we should not add more tokens, unless I'm missing something.

File: utils/utils.ts [hunk 4]
```diff
@@ -331,11 +331,12 @@ export async function getThirdPartyRewards(
   rewardToken: BN,
 ) {
   const api = getApi();
-  return await api.rpc.pos.calculate_3rdparty_rewards_amount(
+  const calculation = await api.rpc.pos.calculate_3rdparty_rewards_amount(
     userAddress,
     liquidityAssetId.toString(),
     rewardToken.toString(),
   );
+  return new BN(calculation.toString());
```
Comment (goncer) [inline utils/utils.ts:339]: use the function stringToBN instead. when large numbers, toString returns a 0x, and new BN will fail.

**Review by goncer — CHANGES_REQUESTED** [review]

File: test/parallel/proffOfStake.3rdPartyRewards.activation.test.ts (hunk 1 above)
Comment (AlexChetverov) [reply test/parallel/proffOfStake.3rdPartyRewards.activation.test.ts]: It was problem on my local node. Change cancelled

File: test/parallel/proffOfStake.3rdPartyRewards.activation.test.ts (hunk 3 above)
Comment (AlexChetverov) [reply test/parallel/proffOfStake.3rdPartyRewards.activation.test.ts]: It was problem on my local node. Change cancelled

File: utils/utils.ts (hunk 4 above)
Comment (AlexChetverov) [reply utils/utils.ts]: Done

**Review by goncer — APPROVED** [review]
